### PR TITLE
bump kube-ui to v3

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
LGTM

File: cluster/addons/kube-ui/kube-ui-rc.yaml [hunk 1]
```diff
@@ -1,27 +1,27 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: kube-ui-v2
+  name: kube-ui-v3
```
Comment (timstclair) [inline cluster/addons/kube-ui/kube-ui-rc.yaml]: I think I was copying another example when I put this together, but now that I know the system better, is there anyreason to include the version in the lables & name like this? I think we could probably just rely on the image version.

**Comment by timstclair** [comment]
LGTM

File: cluster/addons/kube-ui/kube-ui-rc.yaml (hunk 1 above)
Comment (mikedanese) [inline cluster/addons/kube-ui/kube-ui-rc.yaml]: Might be specific to the addon code, but all other addons do this.

**Pull request merged**